Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs

```diff
@@ -57,4 +60,25 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
 
         stationJobs.TrySetJobSlot(station, job, amount, true);
     }
+
+    // autofill for station ID
+    public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
+    {
+        if (args.Length != 1)
+        {
+            return CompletionResult.Empty;
+        }
+
+        var stations = _entityManager
+            .System<StationSystem>()
+            .Stations
+            .Select(station =>
+            {
+                var meta = _entityManager.GetComponent<MetaDataComponent>(station);
+
+                return new CompletionOption(station.ToString(), meta.EntityName);
+            });
```

Comment:
Can't you just use the component completionhelper and get stations?

Reply:
I really don't know, I just ported the code from crewmanifest